LEGO1/lego/legoomni/src/video/legovideomanager.cpp

```diff
@@ -403,40 +403,70 @@ inline void LegoVideoManager::DrawCursor()
 		->BltFast(m_cursorXCopy, m_cursorYCopy, m_cursorSurface, &m_cursorRect, DDBLTFAST_WAIT | DDBLTFAST_SRCCOLORKEY);
 }
 
+// 4x5 bitmap font
+const uint8_t digitFont[5][10] = {
```

Comment:
Can you move new code / functions to the bottom of the compilation unit? That's what we have generally done with new stuff

Reply:
And forward declare it?

Reply:
I'd add them as member functions to the header at the end of the existing function declarations. Just to stay in style with the current code base (which is 95% class/object based):

![image](https://github.com/user-attachments/assets/1fd882d9-5b35-4d69-89d0-5cc50a73b373)